menas/src/main/resources/application.properties.template

```diff
@@ -120,3 +123,11 @@ za.co.absa.enceladus.menas.oozie.splineMongoURL=mongodb://localhost:27017
 #In some cases extra driver options need to be surrounded by double quotes
 #This seems to depend of the version of oozie
 #za.co.absa.enceladus.menas.oozie.sparkConf.surroundingQuoteChar="
+
+#Configure the following two parameters to use oozie impersonation
+#za.co.absa.enceladus.menas.oozie.proxyUser=oozieProxyUser
+#za.co.absa.enceladus.menas.oozie.proxyUserKeytab=/path/to/keytab
+
+#Provide optional spark config options to be passed onto standardization and conformance jobs
+#za.co.absa.enceladus.menas.oozie.extraSparkConfigs={"spark.eventLog.dir" : "hdfs:///spark2-history/", \
+"spark.history.fs.logDirectory": "hdfs:///spark2-history" }
```

Comment:
Only 1st line is commented

Reply:
yeah it's a single config, newline is escaped here

Reply:
Yeah ok cool